graylog2-web-interface/src/components/events/events/EventsRefreshControls.tsx

```diff
@@ -24,9 +24,13 @@ import { getPathnameWithoutId } from 'util/URLUtils';
 import useLocation from 'routing/useLocation';
 import useMinimumRefreshInterval from 'views/hooks/useMinimumRefreshInterval';
 import RefreshControls from 'components/common/RefreshControls';
-import useDefaultInterval from 'views/hooks/useDefaultIntervalForRefresh';
 import AutoRefreshProvider from 'views/components/contexts/AutoRefreshProvider';
 import { useTableFetchContext } from 'components/common/PaginatedEntityTable';
+import type { RefreshConfig } from 'views/components/contexts/AutoRefreshContext';
+import { durationToMS } from 'util/DateTime';
+
+const defaultInterval = 'PT30S';
```

Comment:
I am not sure if we should hardcode it. What are arguments against reusing the search refresh defaults for this? 5s does not seem to be too low for me.

Reply:
@dennisoelkers  30 sec that was a product requirement.

Reply:
Can you clarify if it was even considered to reuse the configurable values or if that number was just an educated guess, without knowing that a user can configure it in other places?

Reply:
I addressed this question. Let's wait for a response

Reply:
@dennisoelkers After discussions we decided to use the default interval from configuration.  Pushed the changes